src/content/changes/highlights/2024-10-10.md

```diff
@@ -0,0 +1,66 @@
+---
+date: 2024-10-10T14:00:00
+title: Highlights for the week ending 2024-10-10
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Observability
+
+- [alloy-gateway-app](https://github.com/giantswarm/alloy-gateway-app) version [0.1.0](https://github.com/giantswarm/alloy-gateway-app/releases/tag/v0.1.0)
+    - Create initial app version.
+
+- [alloy-app](https://github.com/giantswarm/alloy-app) version [0.6.1](https://github.com/giantswarm/alloy-app/compare/v0.5.2...v0.6.1)
```

Comment:
WDYT about this instead of the whole list?
```suggestion
- [alloy-app](https://github.com/giantswarm/alloy-app) version [0.6.1](https://github.com/giantswarm/alloy-app/compare/v0.5.2...v0.6.1)
  - Update our alloy agents to upstream version 1.4.2 and the helm chart to 0.9.1.
  - Fix ciliumnetworkpolicy to allow clustering mode.
```